src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/build_plan/JenkinsBuildPlanService.java

```diff
@@ -178,9 +178,7 @@ private JenkinsXmlConfigBuilder builderFor(ProgrammingLanguage programmingLangua
         }
         return switch (programmingLanguage) {
             case JAVA, KOTLIN, PYTHON, C, HASKELL, SWIFT, EMPTY -> jenkinsBuildPlanCreator;
-            case VHDL -> throw new UnsupportedOperationException("VHDL templates are not available for Jenkins.");
-            case ASSEMBLER -> throw new UnsupportedOperationException("Assembler templates are not available for Jenkins.");
-            case OCAML -> throw new UnsupportedOperationException("OCaml templates are not available for Jenkins.");
+            default -> throw new UnsupportedOperationException(programmingLanguage + " templates are not available for Jenkins.");
```

Comment:
we want to avoid default in switch expressions, because the compiler would not warn us if new enum values are added

Reply:
Corrected in [103f8fa](https://github.com/ls1intum/Artemis/pull/9074/commits/103f8fa2b46e4efb8a960aae6db1dfdc1e8809e6).